app/controllers/assets_controller.rb

```diff
@@ -155,6 +155,13 @@ def view
     render layout: false
   end
 
+  def start_edit
+    asset = Asset.find_by_id(params[:id])
```

Comment:
we could just use the `load_vars` in `before_action` as it is used for other actions as well. Then we should additionally check for permissions if `current_user` even has permissions to add activity.